server/php-runner.h

```diff
@@ -59,46 +58,63 @@ void dump_query_stats();
 
 void init_handlers();
 
+class CoroutineStack : vk::not_copyable {
```

Comment:
Such a name can be confusing. Coroutines in kphp don't have their own stack actually, but use the single common one.
Let's rename it to smth like 'CustomStack' or 'PhpScriptStack'